src/main/java/app/ProgramAndYearForm.java

```diff
@@ -0,0 +1,31 @@
+package app;
+
+/**
+ * Created by Faisal on 2018-03-21.
+ */
+public class ProgramAndYearForm {
```

Comment:
Is this standard practice, to create a form object for each form?

Reply:
To pass a multiple objects for form validation you can either use a form class or a map. I used the form because it was quicker

Reply:
Alright, go to go through with approval.